### ISSUE TITLE
Double-click should select words

### ISSUE THREAD

**Issue by guds-sb** [YoYoGames/GameMaker-Feature-Requests#1228]
### Is your feature request related to a problem?

This is a common feature among many text editors and I'm constantly using it on VSCode, so it would help a lot if GMS2 script editor had this same feature, as it makes programming a bit faster

### Describe the solution you'd like

double-clicking a word in the script editor should select it, stopping at spaces and special characters. Holding the mouse button after the second click should allow me to select snapping at words instead of each character
Windows 10
IDE v2023.6.0.92
Runtime v2023.6.0.139

**Comment by Alphish** [comment]
Just for the record, GM sort of already selects a word on double click, but only after a mouse release.

So it works as such:
- user rapidly clicks, releases, clicks, releases
- then the whole word is selected
- if you just click, release and click, the word isn't selected, and if you move mouse around after the second click, you get regular, non-word snapped selection

In contrast, the VSCode behaviour is like so:
- use rapidly clicks, releases, clicks
- the whole word is selected without needing to release the second time
- if you move the mouse around after the second click, the selection is snapped to the word boundaries

**Comment by L-Clouds** [comment]
Duplicate of #989 I believe
